GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Engines.cfg

```diff
@@ -1143,6 +1143,9 @@
 //  Spin Motor (small)
 //  ==================================================
 
+//Source for separation motors: https://www.northropgrumman.com/wp-content/uploads/NG-Propulsion-Products-Catalog.pdf (Page 49)
+//For dry mass:propellant mass, using a ratio of 1:0.9, a little better than the BSM
```

Comment:
This actually should be a bunch worse, since this is supposed to be a 1940s solid, and the BSM referenced is a 1990s/2000s era solid, probably with a fiberglass casing. I'd suggest using 1.11 for the volume, to preserve the burntime.